package.json

```diff
@@ -40,14 +40,9 @@
     "punycode.es6.js"
   ],
   "scripts": {
-    "test": "mocha tests",
+    "test": "npx mocha tests",
     "build": "node scripts/prepublish.js"
   },
-  "devDependencies": {
-    "codecov": "^3.8.3",
-    "nyc": "^15.1.0",
-    "mocha": "^10.2.0"
-  },
```

Comment:
With these changes, you can no longer `git clone` + `npm install` to get everything you need to work offline. Please revert these changes.

Reply:
Will do. 

Food for thought:

- nyc is almost unsupported (last release on npm, 4 years ago).
- the codecov module is [officially](https://www.npmjs.com/package/codecov) deprecated
- the coverage reporters (codecov, coveralls, etc.) are nudging everyone to move to GHA style processing
- the included GitHub action will report the coverage results in the PRs
- moving the code coverage entirely to a GitHub Action improves the *default* install (`npm -i`), because 99.9% <sup>1</sup> of the time, nobody benefits from installing dev modules they'll never use. I don't know about you but many of my PRs come from folks that still greatly struggle with git, let alone running `npm test`.
- if `mocha` is installed globally on your machine (as may be the case for a developer who prefers it), running `npx mocha` will use the globally installed (via `npm i -g`) version. Not only does this work great offline, but it saves a lot of wasted developer time when doing installs and reinstalls of modules.
- (aside: mocha is my favorite test runner), but removing mocha as a dep pairs nicely with #135 

1. Made up statistic, but lets be honest: so few people used `npm i --production` that all 68 of us noticed when npm changed it to `npm i --omit=dev`